.github/workflows/release.yml

```
@@ -0,0 +1,69 @@
+    permissions:
+      contents: write
+      pull-requests: write
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: 'Setup'
+        uses: ./.github/actions/setup
```

actionlint (1.7.4)

description is required in metadata of "Setup" action at "/home/jailuser/git/.github/actions/setup/action.yml"

(action)